### ISSUE TITLE
build instructions out of date

### ISSUE THREAD

**Issue by flowchartsman** [augustoroman/v8#8]
```
~/go/src/github.com/augustoroman/v8 $ ls include
APIDesign.md                  v8-inspector.h                v8-version-string.h
DEPS                          v8-platform.h                 v8-version.h
OWNERS                        v8-profiler.h                 v8.h
PRESUBMIT.py                  v8-testing.h                  v8config.h
libplatform                   v8-util.h
v8-inspector-protocol.h       v8-value-serializer-version.h
~/go/src/github.com/augustoroman/v8 $ ls libv8/
libv8_libbase.a     libv8_libplatform.a
~/go/src/github.com/augustoroman/v8 $ go build
# github.com/augustoroman/v8
ld: library not found for -lv8_base
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by flowchartsman** [comment]
I think this is because the build instructions in README.MD are out of date and don't jive with what symlink.sh expects.  I'll see if I can get everything working and update the docs with a PR.

**Comment by augustoroman** [comment]
Fixed by #9
